### DISCUSSION TITLE
EclipseCon 2022 and RDF4J

### DISCUSSION THREAD

**Discussion by kenwenzel** [eclipse-rdf4j/rdf4j#3845] [General]
Dear community,
does someone plan to participate in this year's EclipseCon and spread the word about RDF4J and related projects?

https://www.eclipsecon.org/2022/talk-proposal-topics

Best regards,
Ken

**Comment by hmottestad** [comment]
I don't have any plans at least. I didn't see any graph related topic for promoting RDF4J as a tool. 

There is a general open source topic, so that would be an option if we want to share about how it is to participate in a medium size open source project. Would be a nice place for you to share your experience with becoming part of project here.

And there is a catch all topic where we could present something specific from the project. Your lmdb backed store would fit in there nicely :)
